src/client/components/pages/about.js

```diff
@@ -110,7 +115,8 @@ function AboutPage() {
 			<p>
 				The core components of BookBrainz are the site, the
 				webservice and the database. All of these components are
-				running on MetaBrainz's Sakura server, which also runs&nbsp;
```

Comment:
I don't mind this change - not part of the changes mentioned in the PR, so generally it would be better to keep it separate (don't worry for now), but it's an improvement nonetheless.